README.md

```diff
@@ -5,33 +5,173 @@ state container for [Reason](https://github.com/facebook/reason) applications.
 Its scope goes beyond that of managing state for a single component, and
 concerns the application as a whole.
 
-## Word of Caution
+## Installation
 
-**[You might not need this library](https://medium.com/@dan_abramov/you-might-not-need-redux-be46360cf367)**, especially so in a language which provides good enough construction blocks out of the box. **ReasonReact [already comes with reducers](https://reasonml.github.io/reason-react/blog/2017/09/01/reducers.html)**!
+Install via:
 
-However, in the spirit of unifying the Reason community around an authoritative Redux implementation and to avoid churn, we'll polish Reductive to make it production-ready for those who do wish to use it anyway.
+```shell
+$ npm install --save reductive
+```
 
-## Requirements
+and add it to `bsconfig.json`:
 
-A recent release of [Node](https://nodejs.org/en/) LTS should be sufficient.
+```sh
+"bs-dependencies": ["reductive"]
+```
 
-## Installation
+## Hooks
 
-Installation of this module can be achieved using the `npm` command like so:
+`ReasonReact` version 0.7.0 has added [support](https://reasonml.github.io/reason-react/blog/2019/04/10/react-hooks) for react hooks and `reductive` now includes a set of hooks to subscribe to the store and dispatch actions. With the new hooks there is no need to use `Lense`s that wrap components, which results in simpler and cleaner code with a flat component tree. Moreover, the new hooks API is [safe to use in concurrent mode](https://github.com/facebook/react/tree/master/packages/use-subscription#use-subscription).
 
-```shell
-$ npm install reasonml-community/reductive
+The `Lense` API is still available, since there is support for the old `jsx` and reducer style components, but can be considered deprecated, since the old `jsx` syntax is also marked as [deprecated](https://reasonml.github.io/reason-react/docs/en/jsx-2) in the `reason-react` docs. The preferred way of using `reductive` is via the new hooks API.
```

Comment:
Good point! Might be cool to add `[@deprecated]` warnings where applicable.

Reply:
Marked the old API as `deprecated`.